### PR TITLE
Code review - Initial CRM App release

### PR DESCRIPTION
**Pull request by amadoukane96**
Could you please review my code guys ? 😄

### CODE REVIEW

**Comment by xhlulu** [comment]
Do you have a link to the app?

File: app.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import dash
+import math
```
Comment (xhlulu) [inline app.py]: math should be imported before dash: "Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports."

https://www.python.org/dev/peps/pep-0008/

**Comment by amadoukane96** [comment]
here is the link @xhlulu 😄 
https://dash-gallery.plotly.host/dash-salesforce-crm

File: app.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import dash_core_components as dcc
+import dash_html_components as html
+import pandas as pd
+import flask
+import dateutil.parser
```
Comment (xhlulu) [inline app.py]: Same here

File: app.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+from sfManager import sf_Manager
+
+server = flask.Flask(__name__)
+app = dash.Dash(__name__, server=server)
+app.config.suppress_callback_exceptions = True
+
+sf_manager = sf_Manager()
+
+millnames = ["", " K", " M", " B", " T"] # used to convert numbers
```
Comment (xhlulu) [inline app.py]: Would suggest using all caps when you have a final variable or constant. Read more here: https://www.python.org/dev/peps/pep-0008/#id48

File: app.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+
+# return html Table with dataframe values  
+def df_to_table(df):
```
Comment (xhlulu) [inline app.py]: Love this function! How is it used inside the app.layout? If you are using similar to a component, then I'd suggest you to read more about reusable components: https://dash-workshop.plot.ly/reusable-components

Comment (amadoukane96) [reply app.py]: Thanks man very helpful @xhlulu  👍

**Comment by shammamah** [comment]
Other than the minor PEP8 stuff @xhlulu mentioned, it looks great!

**Pull request merged**